copulas/univariate/gaussian_kde.py

```diff
@@ -180,6 +181,45 @@ def percent_point(self, U):
 
         return X
 
+    def percent_point_fast(self, U):
```

Comment:
Once this is working, the old one is not needed anymore. So let's replace the old one with the new implementation and also get rid of everything related to the old one.

Reply:
There are still some numerical stability issues, I'll remove it after they're resolved.